4_setup_train_ff_topologies/2025_09_03_test_split/setup_train_ff_topo.py

```diff
@@ -0,0 +1,562 @@
+"""Prepare force field and topology information for training.
+
```

Comment:
Could you please mention somewhere (e.g. here in the docstring, in the shell script, or a README) that the JSON format is something custom to this script, probably just for debugging, and that you'll need a custom decoder/converter as well to re-create the forcefield?

Reply:
Yeah good idea, thanks